### PR TITLE
i/8034: Change the API and docs for ImageInsert plugin.

### LINKED ISSUE #8034: Fix the naming of the insert image panel
## 📝 Provide a description of the improvement

First mentioned in https://github.com/ckeditor/ckeditor5/issues/7890#issuecomment-685703086.

The proposal is `image.insert.integrations` and sounds fine to me too.

We should do this change as soon after #8019 as possible.

---

If you'd like to see this improvement implemented, add a 👍 reaction to this post.

### PR DESCRIPTION
**Pull request by panr**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

BREAKING CHANGE (image): Changed the integration API for the `ImageInsert` plugin. Closes #8034.

Docs (image): Changed the feature and API docs for the `ImageInsert` plugin. See #8034.

### CODE REVIEW

File: packages/ckeditor5-image/src/imageinsert.js [hunk 1]
```diff
@@ -66,11 +65,11 @@ export default class ImageInsert extends Plugin {
  * See {@link module:core/editor/editorconfig~EditorConfig all editor options}.
  *
  * @protected
- * @interface module:image/imageupload~ImageUploadPanelConfig
+ * @interface module:image/imageinsert~ImageInsertConfig
  */
 
 /**
- * The list of {@link module:image/imageinsert~ImageInsert} integrations.
+ * Image insert panel view configuration contains a list of {@link module:image/imageinsert~ImageInsert} integrations.
```
Comment (godai78) [inline packages/ckeditor5-image/src/imageinsert.js:72]:
```suggestion
 * The image insert panel view configuration contains a list of {@link module:image/imageinsert~ImageInsert} integrations.
```

**Review by godai78 — APPROVED** [review]

File: packages/ckeditor5-image/src/imageinsert.js (hunk 1 above)
Comment (panr) [reply packages/ckeditor5-image/src/imageinsert.js]: Thanks, I used your suggestion here 👌

File: packages/ckeditor5-image/docs/features/image.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ See the {@link features/image-upload Image upload} guide.
 
 Besides the ability to insert images by uploading them directly from your disk or via CKFinder, you can also configure CKEditor 5 to allow inserting images via source URL.
 
-In order to enable this option, install the `ImageInsert` plugin.
+In order to enable this option, install the `ImageInsert` plugin and adding the `imageInsert` button to the toolbar (it replaces the standard `imageUpload` button).
```
Comment (godai78) [inline packages/ckeditor5-image/docs/features/image.md:90]:
```suggestion
In order to enable this option, install the `ImageInsert` plugin and add the `imageInsert` button to the toolbar (it replaces the standard `imageUpload` button).
```

**Review by godai78 — APPROVED** [review]
Grammar fixed; fine otherwise.

**Pull request merged**